### PR TITLE
Rewrite idr-analysis following feedback

### PR DESCRIPTION
**Pull request by manics**
Closes https://github.com/IDR/idr-analysis.openmicroscopy.org/issues/1

Viewing:
```
docker build -t test .
docker run -it --rm -p 8080:80 test
```
http://localhost:8080/

----

Note this includes a proposed "SLA":
> We will do our best to maintain this scratch space across upgrades of the VAE, though in the event of unforeseen issues we may need to delete all scratch space, and you should consider it as suitable for temporary storage only.
> If as part of a planned update we need to delete our existing storage we will give at least one week's notice on <a href="https://github.com/idr-contrib/community/issues"> idr-contrib/community</a>.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -20,25 +22,63 @@ <h2 class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offs
...
+
+                    <div>
+                      <h1 class="text-center">GitHub login</h1>
+                      <p>
+                        If you are interested in developing your own data analysis notebooks on our analysis platform please contact us with your GitHub username.
+                        This will give you more computational resources including scratch storage space than is available through the public access.
+                      </p>
+                      <p>
+                        We will do our best to maintain this scratch space across upgrades of the VAE, though in the event of unforeseen issues we may need to delete all scratch space, and you should consider it as suitable for temporary storage only.
+                        If as part of a planned update we need to delete our existing storage we will give at least one week's notice on <a href="https://github.com/idr-contrib/community/issues"> idr-contrib/community</a>.
```
Comment (joshmoore) [inline src/index.html]: :+1: for community issues. Either for this document or for our internal OPS, I'd _also_ tweet that created issue to IDRstatus.

File: src/index.html [hunk 2]
```diff
@@ -20,25 +22,63 @@ <h2 class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offs
...
 
-                    <h2>Elixir AAI login</h2>
-                        <p>If you have an Elixir AAI account you can use it to access the analysis platform. This is currently a demonstrator, and is identical to the public access. Please note we are unable to investigate authentication errors between your chosen identity provider and Elixir AAI.</p>
-                        <div class="row column text-center"><a href="/vae-aai/hub/oauth_login" class="large button sites-button">Login with Elixir  AAI</a></div>
+                    <hr class="whitespace">
 
-                    <h2>More information</h2>
-                        <p>For more information about the IDR please see the <a href="https://idr.openmicroscopy.org/">IDR home page</a>.</p>
+                    <div>
+                      <h1 class="text-center">Elixir AAI login</h1>
```
Comment (joshmoore) [inline src/index.html:68]: I keep going back and forth on whether to have `(experimental)` in the title of this one...

File: src/deployment.html [hunk 3]
```diff
@@ -9,24 +9,49 @@
                 <div class="medium-12 columns">
                     <h1 class="hero-main-header">Deployment</h1>
                     <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
-                    TODO</p>
+                      The VAE runs on a Kubernetes Cluster hosted on an <a href="https://www.openstack.org/">OpenStack</a> cloud at <a href="https://www.ebi.ac.uk/">EMBL-EBI</a>.
```
Comment (joshmoore) [inline src/deployment.html]: Another URL to potentially offer: http://www.embassycloud.org/

File: src/deployment.html [hunk 4]
```diff
@@ -9,24 +9,49 @@
...
                 </div>
             </div>
         </div>
-        <!-- end Deployment section -->
+        <!-- end Kubernetes section -->
+
+        <!-- begin JupyterHub section -->
+        <hr class="whitespace">
+        <div class="row column text-center">
+            <h2>Kubernetes cluster</h2>
```
Comment (joshmoore) [inline src/deployment.html]: Dupe?

**Review by joshmoore — COMMENTED** [review]
Few misc. comments added, but in general, it looks quite nice. My biggest issues which can be handled in a follow-up are:

 * should the header menus match idr.o.org?
 * e.g. how does one navigate back to the main page after going to deployment.html?
 * Deployment is now a bit anemic. Do you see adding more or should we combine the 2 pages?

**Comment by manics** [comment]
Updated.
- I think we can sort out the menus once we know whether changes are needed in https://github.com/IDR/idr.openmicroscopy.org for IDR-0.5.0.
- I think we should keep deployment separate even if it's fairly minimal. The front page should focus on what we present to the public.

**Comment by manics** [comment]
Another thought: shall I add something like "your scratch space may be deleted if you have not used the service for more than a month (or 2)" to give us more flexibility if needed?

**Comment by joshmoore** [comment]
> shall I add something like "your scratch space may be deleted if

At the moment, it's being deleted more than once a month anyway, so seems fine to start with that. A review will need to be in place though for any idr-analysis redeploy that would delete the scratch space.

> I think we can sort out the menus once we know whether changes are needed in 

Happy for it to be a follow-up, but if we're releasing prod50 next week this will have to be a quick follow-up.

**Comment by manics** [comment]
Updated. The main motivation is in case we get an unexpectedly large number of one-off users resulting in us running short on space.

**Review by joshmoore — APPROVED** [review]
Thanks. Let's get this in and we can re-review in the final PR later this week.

**Pull request merged**